src/scss/main.scss

```
@@ -9,3 +9,7 @@ p {
     }
   }
 }
+
+h1, h2, h3, h4, h5, h6 {
```

Since all headlines are capitalized anyway we can add here I guess.

```suggestion
h1, h2, h3, h4, h5, h6 {
text-transform: uppercase;
```

---

Done!